src/main/java/random_csapatnev/GraphicsCharacter.java

```diff
@@ -15,8 +15,11 @@
  */
 public abstract class GraphicsCharacter extends JPanel implements IGraphics 
 {
-	int x=0,y=0,width=0,height=0;
-	JPanel parent = null;
+	int graphicsx = 0;
```

Comment:
Camelcase jobban olvasható lenne szerintem.

Reply:
Az alatta lévő adattagoknál is.

Reply:
[c737dc6](https://github.com/BME-MIT-IET/iet-hf-2023-grafadatbazis/pull/14/commits/c737dc62923c8571e82fc37e8ab331190121efff) commitban javítva.